sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/PythonUDF.scala

```diff
@@ -51,6 +51,27 @@ object PythonUDF {
     // support new types in the future, e.g, N -> N transform.
     e.isInstanceOf[PythonUDAF]
   }
+
+  def correctEvalType(udf: PythonUDF): Int = {
+    if (udf.evalType == PythonEvalType.SQL_ARROW_BATCHED_UDF) {
+      if (conf.pythonUDFArrowFallbackOnUDT &&
```

Comment:
Can we simply get `conf.pythonUDFArrowFallbackOnUDT` as a parameter? It looks too much for me to bring `SQLConfHelper` simply in order to use `conf` in this object. WDYT, @peter-toth ?